packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -212,7 +245,8 @@ function mergeConfig(options, context, gitRawCommitsOpts, parserOpts, writerOpts
 
           context = _.assign({
             issue: hostOpts.issue,
-            commit: hostOpts.commit
```

Comment:
I wish the style rules for this repository allowed for trailing commas so that these lines wouldn't need to be touched to add that comma for a new line underneath. (Just my thoughts on the matter. Nothing actually needs to change here)